quodlibet/quodlibet/plugins/__init__.py

```diff
@@ -500,3 +500,69 @@ def ConfigCheckButton(cls, label, name, default=False):
             cls.config_set(name, default)
         return ConfigCheckButton(label, PM.CONFIG_SECTION,
                                  option, populate=True)
+
+
```

Comment:
Unless I've missed the point, this seems like a lot of OOP to do what `PluginConfigMixin` is already doing (`config_get_bool`, `config_set` etc)?